components/customs/SectionHeader.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faInfoCircle } from "@fortawesome/free-solid-svg-icons";
+
+//interface of expected props
+interface SectionHeaderProps {
+  header_text: string;
+  tooltip_text: string;
+}
+
+const SectionHeader: React.FC<SectionHeaderProps> = ({
+  header_text,
+  tooltip_text,
+}) => {
+  const [showTooltip, setShowTooltip] = useState(false);
```

Comment:
Can we use the shadcn tooltip here? https://ui.shadcn.com/docs/components/tooltip

Reasoning: Shadcn and other ui libraries (headless, material, chakra, mantine) etc. all have the benefit of having a lot of small things preset. Accesssibility, styling, themeing etc. So we want to use them as much as possible. Always look at the shadcn website to check if there exists a componenent

I specifically really like shadcn because you can customize the components top to bottom which gives us full control over the functionality etc and we do not need to create wrappers. Similar feeling for Headless UI.

Shadcn itself is built on radix so that would be the second place to check.

Reply:
I have changed it to use shadcn and it works with the proper tool tip text showing up. I have encountered some issues attempting to change the styling such as the background color and width are not being applied. I did use the manual way to setup with the src code places inside the components/ui folder. I am currently unsure why the styling is not being applied.